### PR TITLE
Update CORS Config

### LINKED ISSUE #37: Fix cors issue

### PR DESCRIPTION
**Pull request by javier-aguilar**
## What functionality does this accomplish?
- Add title to event view 
- Add CORS config to allow requests from other domains
## Current Test Suite: **Mix/ExUnit**
16 tests, 0 failures
### Test Coverage Percentage: 51.2%
- [x] No tests have been changed
- [ ] Some tests have been changed:
- [ ] All of the tests have been changed:
## Checklist:
- [x] I have reviewed my code
- [ ] I have annotated or commented my code, particularly in hard-to-understand areas
- [ ] I have fully tested my code
- [ ] I have partially tested my code (please explain why):
## Related Issues:
Resolves #37 
## Helpful Resources:
- https://medium.com/@yagoazedias/how-to-configure-cors-in-your-phoenix-application-5ef0234bc25f

### CODE REVIEW

File: lib/drag_time_api_web/router.ex [hunk 1]
```diff
@@ -2,6 +2,7 @@ defmodule DragTimeApiWeb.Router do
   use DragTimeApiWeb, :router
 
   pipeline :api do
+    plug CORSPlug, origin: "*"
```
Comment (collinkallery) [inline lib/drag_time_api_web/router.ex:5]: Guessing this solves our problem we brought up a couple hours ago?

File: lib/drag_time_api_web/views/event_view.ex [hunk 2]
```diff
@@ -13,6 +13,7 @@ defmodule DragTimeApiWeb.EventView do
   def render("create.json", %{event: event}) do
     %{type: "event",
       id: event.id,
+      title: event.title,
```
Comment (collinkallery) [inline lib/drag_time_api_web/views/event_view.ex:16]: Cool thanks for adding this in!

File: mix.lock [hunk 3]
```diff
@@ -1,6 +1,7 @@
 %{
   "certifi": {:hex, :certifi, "2.5.2", "b7cfeae9d2ed395695dd8201c57a2d019c0c43ecaf8b8bcb9320b40d6662f340", [:rebar3], [{:parse_trans, "~>3.3", [hex: :parse_trans, repo: "hexpm", optional: false]}], "hexpm", "3b3b5f36493004ac3455966991eaf6e768ce9884693d9968055aeeeb1e575040"},
   "connection": {:hex, :connection, "1.0.4", "a1cae72211f0eef17705aaededacac3eb30e6625b04a6117c1b2db6ace7d5976", [:mix], [], "hexpm", "4a0850c9be22a43af9920a71ab17c051f5f7d45c209e40269a1938832510e4d9"},
+  "cors_plug": {:hex, :cors_plug, "1.5.2", "72df63c87e4f94112f458ce9d25800900cc88608c1078f0e4faddf20933eda6e", [:mix], [{:plug, "~> 1.3 or ~> 1.4 or ~> 1.5", [hex: :plug, repo: "hexpm", optional: false]}], "hexpm", "9af027d20dc12dd0c4345a6b87247e0c62965871feea0bfecf9764648b02cc69"},
```
Comment (collinkallery) [inline mix.lock:4]: This might as well be Japanese to me

**Review by collinkallery — APPROVED** [review]

**Review by atkinson-daniel — APPROVED** [review]
Good catch on this 🐞

File: lib/drag_time_api_web/router.ex (hunk 1 above)
Comment (TravisBorgsmiller) [reply lib/drag_time_api_web/router.ex:5]: It looks like the asterisk allows anybody to access our API

Comment (javier-aguilar) [reply lib/drag_time_api_web/router.ex:5]: Yeah! It allows for other domains outside of localhost:4000 to get data

**Pull request merged**